pkg/operation/seed/seed.go

```diff
@@ -1420,3 +1459,69 @@ func deletePriorityClassIfValueNotTheSame(ctx context.Context, k8sClient client.
 
 	return client.IgnoreNotFound(k8sClient.Delete(ctx, pc))
 }
+
+func getMandatoryExposureClassHandlerLabels(name string) map[string]string {
+	return map[string]string{
+		v1alpha1constants.GardenRole:                    v1alpha1constants.GardenRoleExposureClassHandler,
+		v1alpha1constants.LabelExposureClassHandlerName: name,
+	}
+}
+
+func cleanupOrphanExposureClassHandlerResources(ctx context.Context, c client.Client, exposureClassHandlers []config.ExposureClassHandler, logger *logrus.Entry) error {
+	var exposureClassHandlerNamespaces = &corev1.NamespaceList{}
```

Comment:
Nit for consistent style: Use `<name> := &corev1.NamespaceList{}` (e.g., same below for similar usage)